src/bin/index.js

```diff
@@ -94,6 +102,120 @@ yargs(hideBin(process.argv))
+    (yargs) => {
+      yargs
+        .command(
+          [
+            'start [mode] [type] [proof-level] [mina-branch] [archive] [sync] [pull]',
+          ],
+          'Start the lightweight Mina blockchain network Docker container.',
+          {
+            mode: {
+              alias: 'm',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetModes,
+              default: 'single-node',
+              description:
+                'Whether to form the network with one node or with multiple network participants.\n"single-node" value will make the network faster.',
+            },
+            type: {
+              alias: 't',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetTypes,
+              default: 'fast',
+              description:
+                'Whether to configure the network to be fast or slower with closer-to-real-world properties.',
+            },
+            'proof-level': {
+              alias: 'p',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetProofLevels,
+              default: 'none',
+              description: '"none" value will make the network faster.',
+            },
+            'mina-branch': {
+              alias: 'b',
+              demand: false,
+              string: true,
+              hidden: false,
+              choices: Constants.lightnetMinaBranches,
+              default: 'rampup',
+              description:
+                'One of the major Mina repository branches the artifacts were compiled against.',
+            },
+            archive: {
+              alias: 'a',
+              demand: false,
+              boolean: true,
+              hidden: false,
+              default: true,
+              description:
+                'Whether to start the Mina Archive process and Archive Node API application within the Docker container.',
+            },
+            sync: {
+              alias: 's',
+              demand: false,
+              boolean: true,
+              hidden: false,
+              default: true,
+              description:
+                'Whether to wait for the network to reach the synchronized state.',
+            },
+            pull: {
+              alias: 'u',
+              demand: false,
+              boolean: true,
+              hidden: false,
+              default: true,
+              description:
+                'Whether to pull the latest version of the Docker image from the Docker Hub.',
+            },
+          },
+          async (argv) => await startLightnet(argv)
+        )
+        .command(
+          ['stop [save-logs] [clean-up]'],
+          'Stop the lightweight Mina blockchain network Docker container and perform the clean up.',
```

Comment:
```suggestion
          'Stop the lightweight Mina blockchain network Docker container and perform the logs cleanup.',
```

Reply:
what is the advantage/disadvantage of savings logs vs cleaning logs? (do we have a default log truncate or cleanup?)

Reply:
Cleaning up is not about the logs.

> Whether to remove the Docker container, dangling Docker images, consumed Docker volume, and the blockchain network configuration.